### PR TITLE
Update ldap_proxy_cache_server_setup.adoc

### PR DESCRIPTION
**Pull request by mmattel**
Improvements:
- putting the second ldapsearch command also into syntax highlighting
- adding a table for the option description
- text and highlighting improvements

### CODE REVIEW

**Comment by mmattel** [comment]
@butonic I have improved and updated the structure, and content of the ldap setup.
The old description did not reflect the new ownCloud GUI.
Can you take a look on pls?

File: modules/admin_manual/pages/configuration/ldap/ldap_proxy_cache_server_setup.adoc [hunk 1]
```diff
@@ -34,25 +37,25 @@ There’s not that much to it, just the following five steps:
 [[install-openldap]]
 === Install OpenLDAP
 
-There are a number of
-https://en.wikipedia.org/wiki/List_of_LDAP_software[LDAP server
-implementations] available. The one used in this guide is
-https://en.wikipedia.org/wiki/OpenLDAP[OpenLDAP].
+There are a number of {list-of-ldap-url}[LDAP server implementations] available. +
+The one used in this guide is {open-ldap-url}[OpenLDAP].
```
Comment (settermjd) [inline modules/admin_manual/pages/configuration/ldap/ldap_proxy_cache_server_setup.adoc]: These two sentences should run-together as a single paragraph.

**Comment by mmattel** [comment]
@butonic ping, can you pls have a look on the changes

**Comment by settermjd** [comment]
I made a series of changes to the text and related files in 3a9e83b.

**Comment by settermjd** [comment]
@butonic, can you review pls?

**Pull request merged**